token/inc/token.h

```diff
@@ -7,6 +7,10 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+#define TOKEN_MAJOR_VERSION 0
+#define TOKEN_MINOR_VERSION 1
+#define TOKEN_PATCH_VERSION 0
+
```

Comment:
Just to be clear, when we're ready for this (https://github.com/solana-labs/solana-program-library/pull/47), I just need to build token to update these, right?